File: Colleen/Colleen.c
#include <stdio.h>
/*
    Comment 1
*/
void useless(){}
int main(){char*a="#include <stdio.h>%c/*%c    Comment 1%c*/%cvoid useless(){}%cint main(){char*a=%c%s%c;%c/*%c    Comment 2%c*/%cprintf(a,10,10,10,10,10,34,a,34,10,10,10,10,10);return(0);}%c";
/*
    Comment 2
*/
printf(a,10,10,10,10,10,34,a,34,10,10,10,10,10);return(0);}

File: bonus/Colleen/Colleen.py
#Comment 1
def useless():
	#Comment 2
	return
a = "#Comment 1%cdef useless():%c%c#Comment 2%c%creturn%ca = %c%s%c%cprint a %c (10,10,9,10,9,10,34,a,34,10,37)"
print a % (10,10,9,10,9,10,34,a,34,10,37)
